openstack/compute/v2/extensions/aggregates/results.go

```diff
@@ -78,18 +78,22 @@ type aggregatesResult struct {
 	gophercloud.Result
 }
 
-type CreateResult struct {
-	aggregatesResult
-}
-
-func (r CreateResult) Extract() (*Aggregate, error) {
+func (r aggregatesResult) Extract() (*Aggregate, error) {
 	var s struct {
 		Aggregate *Aggregate `json:"aggregate"`
 	}
 	err := r.ExtractInto(&s)
 	return s.Aggregate, err
 }
 
+type CreateResult struct {
+	aggregatesResult
+}
+
+type GetResult struct {
+	aggregatesResult
```

Comment:
There appear to be some missing fields in the `aggregateResult` struct that can be returned in the body on a GET and POST `deleted_at` and `deleted` are in both. This may have been discussed in the review of POST, but I'll add this comment until I can check.

Reply:
@dklyle Thank you for pointing that out. The omission of those two fields is on me:

https://github.com/gophercloud/gophercloud/pull/691#issuecomment-354357122

Maybe I misunderstood why those fields are marked as deprecated? As well, I suppose it's beneficial to add them for older clouds.

Reply:
@jtopjian I may have missed the deprecation notice. I don't see them flagged in the api docs. But I am known to miss things.

Reply:
They're marked as deprecated at the top of https://github.com/openstack/nova/blob/master/nova/objects/aggregate.py#L38 but in hindsight, it's very possible that is meant for some internal processing of those fields.

We can definitely add these fields in. It might be best to do it in a separate PR for cleanliness.

Reply:
@jtopjian Are there any other "requests for changes"? I could add these fields next PR.